rfd/0012-teleport-versioning.md

```diff
@@ -91,6 +91,29 @@ versions. For example:
 - `v7.*.*`
 - `v6.*.*`
 
+### Git branches
+
+Teleport has several branches related to releases:
+- `master` - main development branch
```

Comment:
While we are at it, let's switch default branch name from `master` to `main` as well.

Reply:
Why? `master` is hardcoded in a zillion places; we’re causing ourselves a headache for no real gain.

For new repos it’s no issue to pick a different name, but for an existing project with 9k stars which many many people are using downstream and have built automation against, I think it causes completely unnecessary disruption.

Reply:
Logically you are absolutely correct, I have no arguments here. However it somehow feels right. I don't think we will solve any worlds problems with this rename. I'm also not going to be upset at the word `master` in the work-appropriate context. But may be we will make a tiny dent in the universe and with numerous of those dents make it better this way? Who knows. I'd give it a go.

Reply:
Sure, updated the RFD and filed https://github.com/gravitational/teleport/issues/5906
It won't be trivial, especially as the team grows in size.